src/datablock/encoding.jl

```diff
@@ -246,7 +246,7 @@ function testencoding(encoding, block, data = mockblock(block))
         @test checkblock(outblock, outdata)
 
         # Test decoding (if supported) works correctly
-        inblock = decodedblock(encoding, outblock, true)
+        inblock = decodedblock(encoding, outblock)
```

Comment:
Why is this changed?

Reply:
For the encodings for which decoding isn't supported yet, like `TabularTransforms` (I'll rename this to TabularPreprocessing), it will give incorrect results I think. 
The encoded block after applying `TabularTransforms` on a `TableRow` is `EncodedTableRow`, and if decoding was supported, then the decoded block should be a `TableRow` again. But as it isn't supported, using the fill option would make it so that `EncodedTableRow` is returned again (instead of `nothing` through the fallback method) which wouldn't be correct. 
The tests after this could fail because `inblock` (which would be the encoded block in this situation), doesn't have to be the same as the original block. Also I'm not sure if the `if` condition would ever be `false` as of now.